gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/GradleWorkarounds.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.dist;
+
+import java.lang.reflect.Proxy;
+import org.gradle.api.DomainObjectCollection;
+import org.gradle.api.GradleException;
+import org.gradle.api.provider.ListProperty;
+
+@SuppressWarnings("UnstableApiUsage")
+final class GradleWorkarounds {
```

Comment:
Could do, but not everyone's off gradle 5 so it might cause unnecessary friction. This is a bit gross but copied wholesale from GCV and we know it works.

Reply:
given that we're approaching 6.4 I think its ok the raise the dep but sure

Reply:
It'd be great to tackle the dep lift in a separate PR, just to help us laggers out a little.